hangman/hangman.py

```diff
@@ -0,0 +1,34 @@
+import random
+import re
+
+
+words = ["Antartica", "Mississippi", "Opulent", "Capricious", "Xenophobe"]
+chosen_word = random.choice(words).upper()
+
+
+def play_hangman(chosen_word):
+    answer = " "
+    guessed = False
+    guessed_letter = []
+    print("Let's play hangman!")
+    print("\n")
+    while not guessed:
+        guess = input("Please guess a letter: ").upper()
+        if len(guess) == 1 and guess.isalpha():
+            if guess not in chosen_word:
+                print(guess, "NO")
+                guessed_letter.append(guess)
+
+            else:
+                guessed_letter.append(guess)
+                guessed_index = [n for n in range(
+                    len(chosen_word)) if chosen_word.find(guess, n) == n]
```

Comment:
Since you're already looping and tracking the index, you don't need the `find` expression
```suggestion
                guessed_index = [n for n in range(
                    len(chosen_word)) if chosen_word[n] == guess]
```

Reply:
Btw, even though list comprehensions are concise and cool to use, it's usually more clear & readable to just use regular loops.

Reply:
Yep, I prefer normal loops for readability.